dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/storage/FeatureStorageConfiguration.kt

```diff
@@ -6,19 +6,26 @@
 
 package com.datadog.android.api.storage
 
+import com.datadog.android.core.configuration.BatchSize
+import com.datadog.android.core.configuration.UploadFrequency
+
 /**
  * Contains the storage configuration for an [FeatureScope] instance.
  * @property maxItemSize the maximum size (in bytes) for a single item in a batch
  * @property maxItemsPerBatch the maximum number of individual items in a batch
  * @property maxBatchSize the maximum size (in bytes) of a complete batch
  * @property oldBatchThreshold the duration (in milliseconds) after which a batch is considered too
+ * @property uploadFrequency the desired upload frequency policy
+ * @property batchSize the desired batch size policy
```

Comment:
minor: maybe we should say, that if not explicitly provided, then value will be taken from the core configuration.